pkg/parsers/gitlab/imports.go

```diff
@@ -1,27 +1,130 @@
 package gitlab
 
 import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/argonsecurity/pipeline-parser/pkg/consts"
 	gitlabModels "github.com/argonsecurity/pipeline-parser/pkg/loaders/gitlab/models"
+	"github.com/argonsecurity/pipeline-parser/pkg/models"
+	"github.com/argonsecurity/pipeline-parser/pkg/utils"
+)
+
+var (
+	gitlabRemotePipelineRegex = regexp.MustCompile(`https://gitlab\.com/(?P<group>[\w-_]+)/(?P<project>.*?)/(?:-/)?raw/(?P<ref>.*?)/(?P<filePath>.*\.ya?ml)`)
 )
 
-// TODO: add ref and project
-func parseImports(include *gitlabModels.Include) []string {
+func parseImports(include *gitlabModels.Include) []*models.Import {
 	if include == nil {
 		return nil
 	}
 
-	imports := []string{}
+	imports := []*models.Import{}
 	for _, item := range *include {
-		if item.Template != "" {
-			imports = append(imports, item.Template)
-		}
-		if item.Local != "" {
-			imports = append(imports, item.Local)
-		}
-		if item.Remote != "" {
-			imports = append(imports, item.Remote)
-		}
-
+		imports = append(imports, parseIncludeItem(item))
```

Comment:
This function might return `nil`. 
Maybe it is better to check it first:
```suggestion
                 if importItem := parseIncludeItem(item); importItem != nil {
		        imports = append(imports, importItem)
		}
```